src/test/java/PanbyteTest.java

```diff
@@ -0,0 +1,439 @@
+import com.code_intelligence.jazzer.junit.FuzzTest;
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PanbyteTest {
+
+    //region Provided tests
+    @Test
+    void testFromHexToBytes() {
+        String echo = "74657374";
+        String[] args = {"-f", "hex", "-t", "bytes"};
+        String expectedOutput = "test";
+        String actualOutput = getOutputOfProgramCall(echo, args);
+        assertEquals(expectedOutput, actualOutput);
+    }
+
+    @FuzzTest
```

Comment:
**issue:** only one fuzz test.

This fuzz test does not even run in the pipeline.